### ISSUE TITLE
Home Assistant external resolv KO

### ISSUE THREAD

**Issue by m4dm4rtig4n** [k8s-at-home/charts#1318]
Hello everyone,

For a few days now, I had problems resolving external DNS from Home Assistant, and following a ticket that was sent to me on Github I managed to fix (work around the problem):
https://stackoverflow.com/questions/65181012/does-alpine-have-known-dns-issue-within-kubernetes/65593511#65593511

So I went to the following config and bingo everything (well not all done) works again :
```
dnsConfig:
  options:
    - name: ndots
      value: "1"
```
I recover External DNS, but lost internal resolv in format SERVICE.NAMESPACE.
Bu SERVICE.NAMESPACE.svc.cluster.local work perfectly

What I have trouble understanding is why I had never had the problem before: /

In short, I just wanted to share my experience and maybe understand the cause at the same time

**Comment by onedr0p** [comment]
Check your `/etc/resolv.conf` on the host machine, if you see a search domain there that's likely your issue.

Set a static IP on your nodes and do not have dhcp assign the IP. DHCP adds a search domain into that file.

**Comment by m4dm4rtig4n** [comment]
My `/etc/resolf.conf`:
```
domain MYDOMAIN.fr
search MYDOMAIN.fr.
nameserver 192.168.100.4
```
192.168.100.4 it's my main DNS Resolver host on k3s.

I have try to remove `domain` & `search` and restore `ndots:5` but "resolv" is always broken.

Afterwards, when I talk about resolution, it's quite weird because if I do a `dig google.fr`, I do have a DNS resolution
And if I do a `curl google.fr`, I have a `google.fr not resolv`

**Comment by onedr0p** [comment]
This isn't an issue with home assistant, you have networking problems which is why I suggested to remove the search domains from you `resolv.conf`.

Please come join our discord or as for help on our GitHub discussion forums.
